backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/i18n/po/POTrlRepository.java

```diff
@@ -276,6 +277,15 @@ public void onTrlRecordChanged(@NonNull final PO trlRecord)
 			return;
 		}
 
+		final boolean isUpdatedColumnPresent = Optional.ofNullable(POInfo.getPOInfo(baseTableName))
+				.map(poInfo -> poInfo.getColumnIndex("Updated") > -1)
+				.orElse(false);
+
+		if (isUpdatedColumnPresent)
+		{
+			sqlSet.append(", Updated").append("=").append(DB.TO_DATE(SystemTime.asTimestamp(), false));
```

Comment:
for the records, we have to do that because in `update_ad_element_on_ad_element_trl_update` we do:

![image](https://github.com/metasfresh/metasfresh/assets/1244701/7e450d28-bcb4-4fa8-92ee-c7b32ff7f990)